core/errors/desugar.h

```diff
@@ -13,6 +13,7 @@ constexpr ErrorClass UnnamedBlockParameter{3007, StrictLevel::Strict};
 constexpr ErrorClass UndefUsage{3008, StrictLevel::Strict};
 constexpr ErrorClass UnsupportedRestArgsDestructure{3009, StrictLevel::True};
 constexpr ErrorClass CodeInRBI{3010, StrictLevel::False};
+constexpr ErrorClass DuplicatedHashKeys{3011, StrictLevel::False};
```

Comment:
What do you think about editing `docs/error-reference.md` for this error code to suggest that "if you're trying to write a sig for `_, _` that you need to choose better names, like `_1, _2`?